lib/index.js

```diff
@@ -6,26 +6,30 @@ if (typeof window !== 'undefined' && window) {
 	if (window.jQuery) {
 	  $ = window.jQuery;
 	} else {
-		$ = require('jquery')
+		$ = require('jquery');
 		window.jQuery = $;
 	}
 } else {
   $ = require('jquery');
 }
 
-var BS = require('bootstrap');
 var React = require('react');
 var objectAssign = require('object-assign');
 var getOptions = require('./get-options.js');
 var bsMultiselect = require('./bootstrap-multiselect.js');
 var bsDropdown;
-
+var BS;
 // make it play nice when we already have bootstrap dropdown loaded.
-if (typeof BS === 'undefined' || typeof BS.dropdown === 'undefined') {
-	bsDropdown = require('./bootstrap-dropdown.js');
+try {
+	BS = require('bootstrap');
+	if (typeof BS.dropdown !== 'undefined') {
+    bsDropdown = BS.dropdown;
+	}
+} catch (e) {
+	console.log('No bootstap');
```

Comment:
The console.log shouldn't be necessary- not having bootstrap is supported & handled below